HopsanGUI/Widgets/AnimationWidget.cpp

```diff
@@ -183,6 +183,7 @@ AnimationWidget::AnimationWidget(QWidget *parent) :
     //mpContainer->collectPlotData();
     mpPlotData = mpContainer->getLogDataHandler();
     mpPlayButton->setDisabled(mpPlotData->isEmpty());
+    mpPlayRealTimeButton->setDisabled(mpPlotData->isEmpty());
```

Comment:
Hmm, it feels like this is not the actual solution to the problem, because why does "real-time" animation require that at least one logdata generation exiats? It should not need that I assume. But probably some code assumes that logdata allways exists.

Reply:
Real-time animation itself does not require any logdata. The problem is real-time plotting, i.e. when you double-click on a scope during real-time animation. This creates temporary variables and inserts them into the current generation. If there is no current generation, it will crash.

Reply:
But I agree that this is not the actual solution. It would be better if you could create a new empty  generation if it does not exist.

Reply:
I will merge this PR now to prevent crashes, and then make a separate PR about the actual log generation problem.